.doc_gen/metadata/sts_metadata.yaml

```diff
@@ -0,0 +1,169 @@
+sts_AssumeRole:
+  title: Assume a role with &STS; using an &AWS; SDK
+  title_abbrev: Assume a role
+  synopsis: assume a role with &STS;.
+  category:
+  languages:
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/sts/sts_temporary_credentials
+          sdkguide:
+          excerpts:
+            - description: Assume an &IAM; role and use temporary credentials to
+                list &S3; buckets for the account.
+              snippet_tags:
+                - iam.python.assume_role.complete
+            - description: Assume an &IAM; role that requires an MFA token and use
+                temporary credentials to list &S3; buckets for the account.
+              snippet_tags:
+                - python.example_code.sts.Scenario_AssumeRoleMfa_list_buckets
+  services:
+    sts: {AssumeRole}
+sts_GetSessionToken:
+  title: Get a session token with &STS; using an &AWS; SDK
+  title_abbrev: Get a session token
+  synopsis: get a session token with &STS; and use it to perform a service action that
+    requires an MFA token.
+  category:
+  languages:
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/sts/sts_temporary_credentials
+          sdkguide:
+          excerpts:
+            - description: Get a session token by passing an MFA token and use
+                it to list &S3; buckets for the account.
+              snippet_tags:
+                - python.example_code.sts.Scenario_SessionTokenMfa_list_buckets
+  services:
+    sts: {GetSessionToken}
+sts_Scenario_AssumeRole:
+  title: Assume an &IAM; role with &STS; using an &AWS; SDK
+  title_abbrev: Assume an &IAM; role
+  synopsis: assume an &IAM; role with &STS; and use temporary security credentials to
+    access an &AWS; resource.
+  category: Scenario
+  languages:
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/sts/sts_temporary_credentials
+          sdkguide:
+          excerpts:
+            - description: Create an &IAM; user and a role that grants permission
+                to list &S3; buckets. The user has rights only to assume the role.
+              snippet_tags:
+                - python.example_code.sts.Scenario_AssumeRole_setup
+            - description: Show that listing &S3; buckets with the user's plain
+                credentials is not allowed.
+              snippet_tags:
+                - python.example_code.sts.Scenario_AssumeRole_access_denied
+            - description: Assume the role that grants permission to list &S3; buckets
+                and show that buckets can now be listed.
+              snippet_tags:
+                - iam.python.assume_role.complete
+            - description: Destroy the resources created for the demo.
+              snippet_tags:
+                - python.example_code.sts.Scenario_AssumeRole_teardown
+            - description: Demonstrate how to use the previously defined functions.
```

Comment:
'Demonstrate how to use the previously defined functions.' reads like the user can use this code to demonstrate how to use the previously defined functions. So maybe lead with an action verb - like 'Use/Run the previously defined functions.'? Just a suggestion. 
(I know you've used this syntax in similar contexts in this PR (and in previously submitted PRs); so whatever you decide, make it consistent throughout.)

Reply:
Makes sense. I went with "Run this scenario by using the previously defined functions."